sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/HiveExplainSuite.scala

```diff
@@ -77,7 +77,7 @@ class HiveExplainSuite extends QueryTest with SQLTestUtils with TestHiveSingleto
       "src")
   }
 
-  test("SPARK-6212: The EXPLAIN output of CTAS only shows the analyzed plan") {
+  test("SPARK-17230: The EXPLAIN output of CTAS only shows the analyzed plan") {
```

Comment:
When we backporting this PR: https://github.com/apache/spark/pull/14797 , it breaks the existing test case. It does not correctly work in the master branch. If we do not want to optimize the query of CTAS, we should see `SubqueryAlias`. Thus, the test case did not fail in the master branch.